stdlib/src/builtin/io.mojo

```diff
@@ -89,8 +89,8 @@ struct _fdopen:
 
 
 @no_inline
-fn _flush():
-    with _fdopen(_fdopen.STDOUT) as fd:
+fn _flush(file: Int = _fdopen.STDOUT):
```

Comment:
Once we added [the named comments mentioned earlier](https://github.com/modularml/mojo/pull/2457/files#r1586789342), we should update this to:

```suggestion
fn _flush(file: Int = sys.stdout):
```

and do similarly in the other new `file` arguments.

Reply:
done.